test/E2ETests/E2EApps/E2EApp/E2EApp.csproj

```diff
@@ -1,26 +1,39 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
-  <PropertyGroup>
+  <PropertyGroup Condition="$(DotnetVersion) != 'netfx'">
     <TargetFramework>net7.0</TargetFramework>
     <AzureFunctionsVersion>v4</AzureFunctionsVersion>
   </PropertyGroup>
 
-  <PropertyGroup>
+  <PropertyGroup Condition="$(DotnetVersion) == 'netfx'">
+    <TargetFramework>net48</TargetFramework>
+    <AzureFunctionsVersion>v4</AzureFunctionsVersion>
+  </PropertyGroup>
+    
+    <PropertyGroup>
     <OutputType>Exe</OutputType>
     <_FunctionsSkipCleanOutput>true</_FunctionsSkipCleanOutput>
     <AssemblyName>Microsoft.Azure.Functions.Worker.E2EApp</AssemblyName>
     <RootNamespace>Microsoft.Azure.Functions.Worker.E2EApp</RootNamespace>
     <Nullable>disable</Nullable>
   </PropertyGroup>
 
-  <ItemGroup>
+  <!-- Remove  any .cs files which are not expected to work with net framework -->  
+  <ItemGroup Condition=" '$(TargetFramework)' == 'net48' ">
+      <Compile Remove="Blob\*.cs; Cosmos\*.cs; EventHubs\*.cs; Queue\*.cs; Table\*.cs"/> 
```

Comment:
Why are these being removed? I think we found this does compile when we are nut multi-targeting.

Reply:
Yeah the build is working fine but I have noticed random failures in tests.

For example - 
- Samples changes that would be needed to make E2E build working with Net48 - [Link](https://github.com/Azure/azure-functions-dotnet-worker/compare/surgupta/netfx-e2etest-setup...surgupta/netfx-test?expand=1)
- Two pipeline runs on the same commitId and DotnetVersion as Net7 - [Run1](https://azfunc.visualstudio.com/Azure%20Functions/_build/results?buildId=152811&view=results), [Run2](https://azfunc.visualstudio.com/Azure%20Functions/_build/results?buildId=152816&view=results).
- Run1 has 1 test failure while Run2 has 2 test failures. (I have also seen upto 4 tests failures in other runs).

With the current approach tests are working fine consistently. I have updated the logic here to replace "Remove files" to "Include files" if that feels better.

```csharp
  <ItemGroup Condition=" '$(TargetFramework)' == 'net48' ">
      <Compile Remove="**\*.cs"/>
      <Compile Include="Timer\*.cs; Http\*.cs; *.cs"/>
  </ItemGroup>
```

I am open to suggestions here.

Reply:
I see. I would stick with the `Remove` style you originally had. Do we have issues tracking the work to make those other tests function in net48?

Reply:
Checked a few more times and realized that there are only 4 tests that are failing or showing flakiness. I have updated the PR to skip those 4 and created this [github issue](https://github.com/Azure/azure-functions-dotnet-worker/issues/1910) to track fix for them. This helped in getting rid of few checks I added to avoid those tests. Let me know if the updated PR looks better or should we consider switching back to the old state.